ci/README.md

```diff
@@ -211,6 +211,26 @@ Once a job has been removed from the required list, it requires two
 consecutive successful nightly test runs before being made required
 again.
 
+### Maintainership
+
+Tests should transition to required status within a reasonable timeframe after
+being added/updated. Without this progression, tests face several challenges
+that can impact the overall quality of the project:
+
+- Maintainers lack motivation to stabilize tests when there's no pressure to
+  make them required
+- Non-required failing tests receive little attention from the community
+- Regressions can slip through when failing tests don't block PR merges
+- Once tests become unstable, it becomes increasingly difficult to restore
+  their reliability
+
+To address these concerns, any test that continues to fail for 10 consecutive
+working days will be considered unmaintained and automatically skipped to
+prevent disruption to the entire workflow chain.
```

Comment:
What is the process to automatically skip a test based on this - it feels like a lot of potential logic for a fairly minor benefit? I'd rather say something like 
> any test that continues to fail for 10 consecutive
working days will be considered unmaintained and may be skipped

as I think it's a manual action and we can't commit to guaranteeing that this happens?

Reply:
Your English is great, but thanks for the update!